src/main/java/Duke.java

```diff
@@ -1,10 +1,248 @@
+            Scanner s = new Scanner(f);
+            while(s.hasNext()){
+                String line=s.nextLine();
+                boolean done=false;
+                String[] sentences=line.split(" \\| ");
+                if (sentences[0].equals("T")){
+                    String answer="todo "+sentences[2];
+                    tasks[sum]=new Todo(answer);
+                    if (sentences[1].equals("1")){
+                        done=true;
+                    }
+                    tasks[sum].setDone(done);
+                    sum++;
+                }
+                else if (sentences[0].equals("D")){
+                    String answer="deadline "+sentences[2]+" /by "+sentences[3];
+                    tasks[sum]=new Deadline(answer);
+                    if (sentences[1].equals("1")){
+                        done=true;
+                    }
+                    tasks[sum].setDone(done);
+                    sum++;
+                }
+                else if (sentences[0].equals("E")){
+                    String answer="event "+sentences[2]+" /at "+sentences[3];
+                    tasks[sum]=new Event(answer);
+                    if (sentences[1].equals("1")){
+                        done=true;
+                    }
+                    tasks[sum].setDone(done);
+                    sum++;
+                }
+                }
+            return sum;
+            }
+        catch (FileNotFoundException e){
+            System.out.println("\n\tThe file does not exist.\n");
+        }
+        return 0;
+    }
+
+    public static void writeFile(Task[] tasks, int sum){
+        try {
+            File f = new File("/Users/tissue/Desktop/CS2113T/ip/data/duke.txt");
+            clearFile();
+            FileWriter fw = new FileWriter(f);
+            for (int i = 0; i < sum; i++) {
+                fw.write(tasks[i].writeToFile()+"\n");
+            }
+            fw.close();
+        }
+        catch (IOException e){
+            System.out.println("\n\tTrouble with IO stream.\n");
+        }
+    }
+
+    public static void clearFile(){
+        try {
+            File f = new File("/Users/tissue/Desktop/CS2113T/ip/data/duke.txt");
+            FileWriter fw = new FileWriter(f);
+            fw.write("");
+            fw.flush();
+            fw.close();
+        }
+        catch (IOException e){
+            System.out.println("\n\tTrouble with IO stream.\n");
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greeting();
+        Scanner in=new Scanner(System.in);
+        String answer;
+        Task[] tasks=new Task[100];
+        int sumList=0;
+        sumList=readFile(tasks);
+        while(true)
```

Comment:
Although not necessary, you can consider the use of a `switch` block here.

Reply:
Ahh, this part is bit hard to use switch block because I am using ".contains()" function for judgement.